lib/node_modules/@stdlib/stats/incr/mprod/lib/main.js

```diff
@@ -42,7 +42,7 @@ var format = require( '@stdlib/string/format' );
 */
 function product( workspace, acc, x ) {
 	// Split the incoming value into a normalized fraction and exponent:
-	frexp( workspace, x );
+	frexp.assign( x, workspace );
```

Comment:
```suggestion
	frexp.assign( x, workspace, 1, 0 );
```

Missing stride and offset.